evil-commands.el

```diff
@@ -4710,6 +4710,18 @@ and redisplay the current buffer there."
   :repeat nil
   (evil-move-window 'below))
 
+;;; Tab commands
+
+(evil-define-command evil-tab-next (&optional arg)
```

Comment:
Nitpick: `arg` should not be optional, since if anyone calls this function non-interactively they are doing something wrong.